kore/src/Kore/Reachability/Claim.hs

```diff
@@ -182,6 +181,7 @@ import Pretty (
     Pretty (..),
  )
 import Pretty qualified
+import Kore.Log.WarnBottom
```

Comment:
Interesting that the auto-formatter lets this slip through... (not sorted alphabetically). :man_shrugging:

Reply:
Possibly because the formatter just did not run yet, the nix materialisation front-ran it?